code/procs/scorestats.dm

```diff
@@ -352,6 +355,33 @@ var/datum/score_tracker/score_tracker
 
 		return jointext(., "")
 
+	//I feel like I may be reinventing the wheel here, but here's a recursive wheel
+	//recursively finds all items of type in container
+	proc/find_items_in(obj/container, type)
```

Comment:
Fairly sure there's a proc that does this already.

Reply:
I thought so but couldn't find it, any idea what it's called? Searching global procs for "find" etc hasn't turned it up.

Reply:
`/mob/proc/get_all_items_on_mob()`